docs/build.md

```diff
@@ -0,0 +1,77 @@
+# Development Env
+
+The EasyReflectometry project consists of two packages; an application and a library. 
+To develop effectively for EasyReflectometry, we work on the `develop` branch of both and then periodically merge to `main` when a release is made. 
+
+With this in mind, the best way to build a development environment for EasyReflectometry is as follows. 
+
+```console
+$ mamba create -y -n easy_refl python=3.9
+$ mamba activate easy_refl
+$ git clone git@github.com:easyScience/EasyReflectometryLib.git
+$ cd EasyReflectometryLib
+$ git checkout develop
+$ pip install -e .
+$ cd ../
+```
+
+Then, to that you are working with your local version of `EasyReflectometryLib`, you want to change line 35 of the `EasyReflectometryApp` `pyproject.toml` file from
+
+```toml
+'EasyReflectometryLib @ git+https://github.com/easyScience/EasyReflectometryLib.git@develop',
+```
+
+to use the directory where you have cloned `EasyReflectometryLib`, i.e.
+
+```toml
+'EasyReflectometryLib @ file:///path/to/EasyReflectometryLib',
+```
+
+With this modification in place, you should then be able to install `EasyReflectometryApp`
+
+```console
+$ git clone git@github.com:easyScience/EasyReflectometryApp.git
```

Comment:
```suggestion
$ git clone https://github.com/easyScience/EasyReflectometryApp.git
```

Reply:
You should also be able to install it via `pip` directly from `GitHub` even without cloning:
`pip install git+https://github.com/easyScience/EasyReflectometryApp`. 

And one can also select any specific branch like this: 
`pip install git+https://github.com/easyScience/EasyReflectometryApp@new_ci`

Reply:
```suggestion
$ git clone git+https://github.com/easyScience/EasyReflectometryApp.git
```